cypress/integration/lesson_page_spec.js

```diff
@@ -0,0 +1,16 @@
+describe('lesson page', () => {
+  it('/lesson loads', () => {
+    cy.visit(
+      '/lessons/react-using-react-error-boundaries-to-handle-errors-in-react-components',
+    )
+    cy.expect('#bmpui-id-204 > .bmpui-image').to.exist
```

Comment:
I'm not crazy about the selectors available to us on this page. Even though they're IDs, they feel fragile, but maybe I'm overthinking it?